examples/connectivity/v4/port2serviceprofileprivate/main.tf

```diff
@@ -42,7 +42,8 @@ resource "equinix_fabric_connection" "sp-private-qinq" {
         uuid= data.equinix_fabric_ports.aside.data.0.uuid
       }
       link_protocol {
-        vlan_s_tag= var.aside_link_protocol_stag
+        type= var.aside_link_protocol_type
+        vlan_tag= var.aside_link_protocol_stag
```

Comment:
Changed the attribute type but the variable name is still in the context of `s_tag`. Should be just `_tag` to match the context in which it's used.